pulp_ansible/tests/functional/api/collection/v2/test_proxy.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+from pulp_smash.pulp3.bindings import monitor_task, PulpTaskError
+
+from pulpcore.client.pulp_ansible import (
+    AnsibleRepositorySyncURL,
+)
+
+
+def _sync_and_assert(
+    remote, ansible_repo, ansible_repo_api_client, ansible_collection_version_api_client
+):
+    body = AnsibleRepositorySyncURL(remote=remote.pulp_href)
+    monitor_task(ansible_repo_api_client.sync(ansible_repo.pulp_href, body).task)
+
+    # Check content is present, but no artifacts are there
+    content_response = ansible_collection_version_api_client.list()
+    assert content_response.count == 1
```

Comment:
Yeah let me fix this, I thought it was for that specific repo. Thanks!

Regarding V3 tests, I don't think they are helpful because the real point of these tests is to assert on the downloaders which is shared by both V2 and V3 tests.